Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -275,9 +275,9 @@ - (CGFloat)vSpacingWithPreviousLayoutItem:(ConversationViewItem *)previousLayout
     OWSAssert(previousLayoutItem);
 
     if (self.interaction.interactionType == OWSInteractionType_UnreadIndicator
-        || previousLayoutItem.interaction.interactionType == OWSInteractionType_UnreadIndicator) {
-        // The unread indicator has its own v-margins.
-        return 0.f;
+        || previousLayoutItem.interaction.interactionType == OWSInteractionType_UnreadIndicator
+        || self.shouldShowDate) {
```

Comment:
I've seen a couple changes lately where you pull the margin logic "up" a level, and I think that makes sense. e.g. the views don't have built in padding, allowing us to coordinate all the spacing at the same level. 👍

Reply:
Glad you like it.